lib/galaxy/webapps/galaxy/controllers/dataset.py

```diff
@@ -297,300 +264,242 @@ def __ok_to_edit_metadata(dataset_id):
+            # attribute editing
+            attribute_inputs = [{
                 'name' : 'name',
                 'type' : 'text',
-                'label': 'Name:',
+                'label': 'Name',
                 'value': data.get_display_name()
-            })
-
-            edit_attributes_inputs.append({
+            }, {
                 'name' : 'info',
                 'type' : 'text',
-                'label': 'Info:',
+                'area' : True,
+                'label': 'Info',
                 'value': data.info
-            })
-
-            edit_attributes_inputs.append({
+            }, {
                 'name' : 'annotation',
                 'type' : 'text',
+                'area' : True,
                 'label': 'Annotation',
                 'value': self.get_item_annotation_str(trans.sa_session, trans.user, data),
                 'help' : 'Add an annotation or notes to a dataset; annotations are available when a history is viewed.'
-            })
-
+            }]
             for name, spec in data_metadata:
                 if spec.visible:
                     attributes = data.metadata.get_metadata_parameter(name, trans=trans)
                     if type(attributes) is form_builder.SelectField:
-                        edit_attributes_inputs.append({
-                            'type': 'select',
-                            'multiple': attributes.multiple,
-                            'optional': attributes.optional,
-                            'name': name,
-                            'label': spec.desc,
-                            'options': attributes.options,
-                            'value': attributes.value if attributes.multiple else [attributes.value]
+                        attribute_inputs.append({
+                            'type'      : 'select',
+                            'multiple'  : attributes.multiple,
+                            'optional'  : attributes.optional,
+                            'name'      : name,
+                            'label'     : spec.desc,
+                            'options'   : attributes.options,
+                            'value'     : attributes.value if attributes.multiple else [attributes.value]
                         })
                     elif type(attributes) is form_builder.TextField:
-                        edit_attributes_inputs.append({
-                            'type': 'text',
-                            'name': name,
-                            'label': spec.desc,
-                            'value': attributes.value,
-                            'readonly': spec.get('readonly')
+                        attribute_inputs.append({
+                            'type'      : 'text',
+                            'name'      : name,
+                            'label'     : spec.desc,
+                            'value'     : attributes.value,
+                            'readonly'  : spec.get('readonly')
```

Comment:
This doesn't violate our linting because we added exceptions for Carl - but I think a lot of us would really prefer if we stuck closer to the flake8 guidelines and not added these extra spaces to align the ``:``. I'm fine merging as is but can you not do this next time?

Reply:
xref: https://www.python.org/dev/peps/pep-0008/#pet-peeves